app/javascript/crayons/MultiInput/__stories__/MultiInput.stories.jsx

```diff
@@ -27,7 +30,8 @@ export const Default = (args) => {
 
 Default.args = {
   placeholder: 'Add an email address...',
-  regex: /([a-zA-Z0-9@.])/,
+  inputRegex: /([a-zA-Z0-9@_.+-])/,
+  validationRegex: /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/,
```

Comment:
A very basic regex that I hope we can improve over time as we begin our first use case for this component. I've not invested too much time on the regex at this point so it may not cover all use cases. The goal was to wrap up the functionality and we can perfect it when we use it in the future.